AirCasting/CreateSession/ConfirmCreatingSessionView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ConfirmCreatingSession.swift
+//  AirCasting
+//
+//  Created by Anna Olak on 22/02/2021.
+//
+
+import SwiftUI
+import CoreLocation
+
+struct ConfirmCreatingSessionView: View {
+    var sessionType = "mobile"
+    var sessionName = "Ania's microphone session"
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 50) {
+            Text("Are you ready?")
+                .font(Font.moderate(size: 24, weight: .bold))
+                .foregroundColor(.darkBlue)
+                
+            VStack(alignment: .leading, spacing: 15) {
+                Text("Your \(sessionType) session \(sessionName) is ready to start gathering data.")
```

Comment:
In the designs (https://projects.invisionapp.com/d/main#/console/19745972/413017502/preview#project_console) the session name is accent blue, could you change it here? :) 
PS. I think I did a similar thing somewhere 😄